.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+on:
+  push:
+    branches:
+      - 'main'
+      - 'versions/*'
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  npm-test:
```

Comment:
Regarding the name, in the future we might also have cth-test if we add the conformance test harness to PR CI, so I think it makes sense to disambiguate like this.